core/src/test/java/io/questdb/test/griffin/DropStatementTest.java

```diff
@@ -41,8 +41,7 @@ public class DropStatementTest extends AbstractGriffinTest {
     /* **
      * DROP can be followed by:
      * - TABLE name [;]
-     * - TABLES name(,name)* [;]
```

Comment:
nit: a question around #3438. We no longer support DROP TABLE [table, ...] syntax, right? AFAIR it used to be supported, yet not documented properly.

Reply:
nope, I can add on demand in a follow up pr.